arthas-api/api/post/prompt/index.js

```diff
@@ -28,11 +28,15 @@ const {
 
 const { prefixInput } = require('arthasgpt/src/utils/prefix');
 
+const __statefulChatFunction = {};
```

Comment:
We can cache the stateful `chat` method per worker, but it is a bit of an antipattern in a stateless API, and when scaling instances there will be multiple (inconsistent) histories depending on which worker is picking up the request.

With `CACHE=true` in the `.env` we already cache the query, prompt, and model responses. For scale, it might be worth the trade-off to keep the `prompt` endpoint entirely stateless, relying on library-level caching for speed, and an application-level solution for chat history/context.